var/spack/repos/builtin/packages/py-quantities/package.py

```diff
@@ -11,20 +11,26 @@ class PyQuantities(PythonPackage):
 
     homepage = "https://python-quantities.readthedocs.org"
     pypi = "quantities/quantities-0.12.1.tar.gz"
-    maintainers("apdavison")
+    maintainers = ["apdavison"]
 
+    version("0.14.1", sha256="efeafffc0c0364f891a9327239cd12496bccb55cd037a6d1bf44de706f722877")
     version("0.13.0", sha256="0fde20115410de21cefa786f3aeae69c1b51bb19ee492190324c1da705e61a81")
     version("0.12.5", sha256="67546963cb2a519b1a4aa43d132ef754360268e5d551b43dd1716903d99812f0")
     version("0.12.4", sha256="a33d636d1870c9e1127631185d89b0105a49f827d6aacd44ad9d8f151f331d8b")
     version("0.12.3", sha256="582f3c7aeba897846761e966615e01202a5e5d06add304492931b05085d19883")
 
     depends_on("python@2.7.0:2.7,3.4:3.7", type=("build", "run"), when="@0.12.3")
     depends_on("python@2.7.0:2.7,3.4:3.8", type=("build", "run"), when="@0.12.4:0.12.5")
-    depends_on("python@3.7:3.10", type=("build", "run"), when="@0.13:")
+    depends_on("python@3.7:3.10", type=("build", "run"), when="@0.13.0:0.13.1")
+    depends_on("python@3.8:3.11", type=("build", "run"), when="@0.14:")
```

Comment:
Where do the upper bounds come from?

Reply:
The upper bounds generally reflect the versions of Python that we run CI testing for. If this is too restrictive, they could be removed.

Reply:
I usually don't add Python version ranges unless:

1. the library is tightly coupled to the CPython API and has separate wheels for each Python version on PyPI, or
2. you know for a fact that newer versions of Python won't work.

We frequently deprecate and remove EOL Python versions so the older versions of this library would have to be removed too.

Reply:
Removed upper bounds for python.